TeamCharles/initial-site
# 117-jsReloadRoute :: Sent response on Post for JS route reload

## Status

**READY**
## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ X] This PR does **NOT** require documentation.
## Description

Sends a response to the ajax call so that the done method can execute and reload the route.
## Resolves Issue Number
#117
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

``` sh
git pull --prune
git checkout 117-jsReloadRoute
dotnet run
```
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
1.  dotnet run
2. Click to cart or products
3. Change user selected and route should reload.
## Impacted Files in Application

List general components of the application that this PR will affect:
- site.js
- UserController.cs
